lm_eval/api/samplers.py

```diff
@@ -47,15 +48,16 @@ def get_context(self, doc, num_fewshot):
                         else self.doc_to_choice(doc)[self.doc_to_text(doc)]
                     )
                     + self.target_delimiter
+                    # doc_to_target
                     + (
-                        self.doc_to_target(doc)[0]
+                        str(self.doc_to_target(doc)[0])
```

Comment:
had to wrap these up in `str` as some answers numbers got converted to `int` in the `Task` processing.